src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/gauge/style_panel.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useCallback } from 'react';
+import { EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { AggGroupNames } from 'ui/vis/editors/default';
+import { SelectOption } from '../../controls/select';
+import { GaugeVisParams } from '../../gauge';
+import { GaugeOptionsInternalProps } from '.';
+
+function StylePanel({
+  aggs,
+  setGaugeValue,
+  stateParams,
+  setValue,
+  vis,
+}: GaugeOptionsInternalProps) {
+  const diasableAlignment =
+    aggs.bySchemaGroup[AggGroupNames.Metrics].length === 1 &&
+    !aggs.bySchemaGroup[AggGroupNames.Buckets];
+
+  const setGaugeType = useCallback(
+    (paramName: 'gaugeType', value: GaugeVisParams['gauge']['gaugeType']) => {
+      const minAngle = value === 'Arc' ? undefined : 0;
```

Comment:
`minAngel` and `maxAngel` play role in building the gauge vis :
![image](https://user-images.githubusercontent.com/31325372/63512522-b8025f80-c4ec-11e9-8766-e3b1cab410b5.png)

Not sure what do you mean by skipping them..

Reply:
I suppose you mean something like that ? : 

![image](https://user-images.githubusercontent.com/31325372/63512956-a9687800-c4ed-11e9-8d90-f562dacef0a9.png)

Reply:
Exactly, that's what I meant - moving the default values into the actual rendering code so we don't have to care about it in the form.

Reply:
This would make sense!
I'll create a PR, maybe someone could verify that this changes would not break a possible configuration issues! (I mean the possibility of creating custom `minAngel` and `maxAngel` in the default config or even in a custom plugin )